src/modules/micrortps_bridge/micrortps_client/microRTPS_client_main.cpp

```diff
@@ -96,7 +96,21 @@ static int parse_options(int argc, char *argv[])
 
 		case 'w': _options.sleep_ms        = strtoul(myoptarg, nullptr, 10);    break;
 
-		case 'b': _options.baudrate        = strtoul(myoptarg, nullptr, 10);    break;
+		case 'b': {
+				int baudrate = 0;
+
+				if (px4_get_parameter_value(myoptarg, baudrate) != 0) {
+					PX4_ERR("baudrate parsing failed");
+				}
+
+				if (baudrate < 9600 || baudrate > 3000000) {
```

Comment:
This is redudant. We already have a check here: https://github.com/PX4/PX4-Autopilot/blob/master/msg/templates/urtps/microRTPS_transport.cpp#L337. Please remove.

Reply:
Fixed, however there's a case for 0 baud rate [here](https://github.com/PX4/PX4-Autopilot/blob/2492fb35e46aa49a5ce29f953be97dcbde95cd95/msg/templates/urtps/microRTPS_transport.cpp#L480) which was caused me to believe there was no checks and allowed the client to start with the missing baud rate variable.